### ISSUE TITLE
crashed when i goto "sdf"

### ISSUE THREAD

**Issue by hyperrealgopher** [hyperrealgopher/waffle#116]
crashed with:

```
waffle: Maybe.fromJust: Nothing
CallStack (from HasCallStack):
  error, called at libraries/base/Data/Maybe.hs:148:21 in base:Data.Maybe
  fromJust, called at src/UI.hs:54:48 in waffle-0.9.0.0-inplace:UI
```

**Comment by hyperrealgopher** [comment]
normal network errors not related to URI format

**Comment by hyperrealgopher** [comment]
Fixed in v0.10.1
